res/values/colors.xml

```diff
@@ -107,4 +107,11 @@
     <color name="default_background_start">#121212</color>
     <color name="default_background_end">#171717</color>
 
+    <array name="user_pic_placeholder_primary">
+        <item>#2bca81</item>
```

Comment:
Let's make these colors match the ones in the design

Reply:
You talking about the color array for the old avatar drawable or the design document?